<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentsTab</class>
 <widget class="QWidget" name="StudentsTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">BACKGROUND-COLOR:rgb(6, 17, 60);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>662</width>
       <height>427</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>844</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Andale Mono</family>
       <pointsize>30</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">BACKGROUND-color:rgb(6, 17, 60);
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="font">
             <font>
              <family>Optima</family>
              <pointsize>15</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 15pt &quot;Optima&quot;;
 border-radius: 5px;
background-color:white;
color: rgb(0, 0, 0);
</string>
            </property>
            <property name="placeholderText">
             <string>search with Id/name/year</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QRadioButton" name="radioButton_prep">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;Optima&quot;;</string>
              </property>
              <property name="text">
               <string>prep</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_first">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;Optima&quot;;</string>
              </property>
              <property name="text">
               <string>first</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_second">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;Optima&quot;;</string>
              </property>
              <property name="text">
               <string>second</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_third">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;Optima&quot;;</string>
              </property>
              <property name="text">
               <string>third</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_forth">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;Optima&quot;;</string>
              </property>
              <property name="text">
               <string>forth</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_all">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;Optima&quot;;</string>
              </property>
              <property name="text">
               <string>all</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QTableWidget" name="tableWidgets_students">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>Optima</family>
                  <pointsize>13</pointsize>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 13pt &quot;Optima&quot;;
color: rgb(0, 0, 0);
	
 border-radius: 9px;
background-color:white;</string>
                </property>
                <property name="editTriggers">
                 <set>QAbstractItemView::NoEditTriggers</set>
                </property>
                <property name="sortingEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QScrollArea" name="scrollArea">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;Optima&quot;;</string>
                </property>
                <property name="widgetResizable">
                 <bool>true</bool>
                </property>
                <widget class="QWidget" name="scrollAreaWidgetContents">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>72</width>
                   <height>432</height>
                  </rect>
                 </property>
                </widget>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>37</width>
                  <height>17</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>Optima</family>
                  <pointsize>13</pointsize>
                  <italic>false</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 13pt &quot;Optima&quot;;
background-color: #FF8C32;
	color: #fff;
	font-weight: bold;
	border-style: solid;
	border-width: 1px;
	border-radius: 17px;
	border-color: #f0742f;
	padding: 8px;
</string>
                </property>
                <property name="text">
                 <string>Add new student</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>37</width>
                  <height>17</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Optima</family>
            <pointsize>17</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">COLOR:WHITE;
font: 700 17pt &quot;Optima&quot;;
BACKGROUND-COLOR:rgb(6, 17, 60)
</string>
          </property>
          <property name="title">
           <string>Student information</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="font">
                  <font>
                   <family>Optima</family>
                   <pointsize>13</pointsize>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 13pt &quot;Optima&quot;;</string>
                 </property>
                 <property name="text">
                  <string>First name:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lbl_fname">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="styleSheet">
                  <string notr="true">font: 13pt &quot;Optima&quot;;</string>
                 </property>
                 <property name="text">
                  <string>Last name:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lbl_lname">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="styleSheet">
                  <string notr="true">font: 13pt &quot;Optima&quot;;</string>
                 </property>
                 <property name="text">
                  <string>Age:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lbl_age">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="styleSheet">
                  <string notr="true">font: 13pt &quot;Optima&quot;;</string>
                 </property>
                 <property name="text">
                  <string>E-mail:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lbl_mail">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="label">
                 <property name="styleSheet">
                  <string notr="true">
font: 13pt &quot;Optima&quot;;</string>
                 </property>
                 <property name="text">
                  <string>phone number:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lbl_phone">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="styleSheet">
                  <string notr="true">font: 13pt &quot;Optima&quot;;</string>
                 </property>
                 <property name="text">
                  <string>Grade:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lbl_grade">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTableWidget" name="tbl_courses">
               <property name="styleSheet">
                <string notr="true">font: 13pt &quot;Optima&quot;;
color: rgb(0, 0, 0);

	 border-radius: 9px;
background-color:white;</string>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_editstd">
               <property name="styleSheet">
                <string notr="true">font: 13pt &quot;Optima&quot;;

background-color: #06113C;
	color: #fff;
	font-weight: bold;
	border-style: solid;
	border-width: 1px;
	border-radius: 17px;
	border-color: #06113C;
	padding: 8px;
</string>
               </property>
               <property name="text">
                <string>Edit</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
